src/components/Error.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useState } from 'react';
+import { ErrorType } from '../types/ErrorType';
+
+type Props = {
+  error: ErrorType | null;
+};
+
+export const Error: React.FC<Props> = ({ error }) => {
+  const [errorMessage, setErrorMessage] = useState(true);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isErrorMessage, setIsErrorMessage] = useState(true);
```